src/common/str-utils.cpp

```diff
@@ -77,6 +77,18 @@ is_blank_line(const char *line, size_t len)
     return true;
 }
 
+bool
+is_base64_line(const char *line, size_t len)
+{
+    for (size_t i = 0; i < len && line[i]; i++) {
+        if (!((line[i] >= 'A' && line[i] <= 'Z') || (line[i] >= 'a' && line[i] <= 'z') ||
```

Comment:
For this purpose there is a table B64DEC in stream-armor.cpp, so check may be done via single lookup.

Reply:
ok

but `B64DEC' is declared as static, so either `is_base64_line()` should be moved to `stream-armor.cpp` and called only there, or `B64DEC` should be exported.

Reply:
As `is_base64_line()` is used only in `stream-armor.cpp` it may be safely moved there.